keras_cv/layers/preprocessing/random_cutout_test.py

```diff
@@ -60,32 +60,13 @@ def test_return_shapes_single_element(self):
     def test_random_cutout_single_float(self):
         self._run_test(0.5, 0.5)
 
-    def test_random_cutout_single_int(self):
```

Comment:
Any reason those tests are removed?

Reply:
Yes - these are testing the behavior of using an `int` as the factor.  This was added by a contributor to support a technique that resembles Randomcutout - but is really not different enough to support (static cutout sizes).  Honestly, it felt very forced and not worth supporting so I removed it.

Reply:
Plus there is evidence that hard coding constants like this is WORSE not better for model performance.